### PR TITLE
Sprinkle some DANGER: UNSAFE warnings over the spec about event bodies

### PR DESCRIPTION
**Pull request by turt2live**
Specs [MSC2801](https://github.com/matrix-org/matrix-doc/pull/2801)
**Based on https://github.com/matrix-org/matrix-doc/pull/3094**

**See https://github.com/matrix-org/matrix-doc/compare/pull/3094/head...pull/3099/head for the real diff.**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -232,6 +232,18 @@ reserved for events defined in the Matrix specification - for instance
 `m.room.message` is the event type for instant messages. Events are
 usually sent in the context of a "Room".
 
+{{% boxes/warning %}}
+Event bodies are considered untrusted data. This means that anyone using
```
Comment (ara4n) [inline content/_index.md:236]:
```suggestion
Event bodies are considered untrusted data. This means that any application using
```

File: content/client-server-api/_index.md [hunk 2]
```diff
@@ -1381,6 +1381,18 @@ opaque string. No changes should be required to support the currently
 available room versions.
 {{% /boxes/warning %}}
 
+{{% boxes/warning %}}
+Event bodies are considered untrusted data. This means that anyone using
```
Comment (ara4n) [inline content/client-server-api/_index.md:1385]:
```suggestion
Event bodies are considered untrusted data. This means that any application using
```

**Review by ara4n — APPROVED** [review]
lgtm, modulo clarifying that it's computers which need to verify event bodies, not humans.

**Comment by turt2live** [comment]
fair point, thanks for catching

**Pull request merged**